### ISSUE TITLE
How can i get the pre-trained model?

### ISSUE THREAD

**Issue by GaryHuangJL** [dlut-dimt/SegMaR#1]
Thanks for the answering

**Comment by dlut-dimt** [comment]
We have uploaded the model

**Comment by dlut-dimt** [comment]
> Thanks for the answering

if you have any problems, please contact this mail Shuilian_Yao@mail.dlut.edu.cn if necessary

**Comment by dlut-dimt** [comment]
> Thanks for the answering

We have uploaded the model

**Comment by ZhouHuang23** [comment]
Hello, would you please release the results on the four COD datasets, namely CAMO, CHAMELEON, COD10K, and NC4K? Thank you very much!

**Comment by visionxiang** [comment]
> > Thanks for the answering
> 
> We have uploaded the model

It seems that there is only a stage-1 pretrained model. It would be appreciated if you could release the final (stage-4) pretrained model. In this way, it is also convenient for other researchers to evaluate and compare your method more fairly. Many thanks.

**Comment by dlut-dimt** [comment]
> It seems that there is only a stage-1 pretrained model. It would be appreciated if you could release the final (stage-4) pretrained model. In this way, it is also convenient for other researchers to evaluate and compare your method more fairly. Many thanks.

Yeah, because each stage requires different dilation maps and distort test images, except for the first stage, just providing the model cannot directly get the test results. It is recommended that you re-run the experiment, and if you have any problems with this experiment, you can send me an email.
